sklearn/linear_model/_base.py

```diff
@@ -640,7 +640,7 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
 
     _parameter_constraints = {
         "fit_intercept": [bool],
-        "normalize": [StrOptions({"deprecated"}, internal={"deprecated"}), bool],
+        "normalize": [Internal(StrOptions({"deprecated"})), bool],
```

Comment:
For developer ergonomics, `Internal` looks very similar to `Interval`. (At a glance, I thought it was `Interval`). What do you think of renaming `Interval` to `Range`?

Reply:
I am +1 for `Hidden`